cartography/intel/aws/cloudtrail.py

```diff
@@ -26,10 +26,13 @@ def get_cloudtrail_trails(
     )
 
     trails = client.describe_trails()["trailList"]
-
-    # CloudTrail multi-region trails are shown in list_trails,
-    # but the get_trail call only works in the home region
     trails_filtered = [trail for trail in trails if trail.get("HomeRegion") == region]
+
+    for trail in trails_filtered:
```

Comment:
Can you move this to a transform function? Mostly because these adjustments happen in the `get` they become tricky to test because the `get` is patched out in integration tests